bundles/org.openhab.binding.onecta/src/main/resources/OH-INF/thing/onecta-account.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onecta"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="account">
+		<label>Account</label>
+		<description>This is the account for which your device(s) are registered at Diakin Onecta.</description>
+
```

Comment:
Can you add semantic equipment tag.
For reference: https://www.openhab.org/docs/developer/bindings/thing-xml.html#thing-semantic-equipment-tags